src/Arduino_LoRaConnectionHandler.h

```diff
@@ -18,8 +18,6 @@
 #ifndef ARDUINO_LORA_CONNECTION_HANDLER_H_
 #define ARDUINO_LORA_CONNECTION_HANDLER_H_
 
-//#ifdef defined(ARDUINO_SAMD_MKRWAN1300) || defined(ARDUINO_SAMD_MKRWAN1310)
```

Comment:
Well, I hope it was really useless ;) The question is - does this code need to be compiled when building the library e.g. a MKR GSM 1400?

Reply:
If the library is compiled for a MKR GSM 1400 (or other non-LoRa boards) the header should not be included by any source file because of this code.
`#if defined(ARDUINO_SAMD_MKRWAN1300) || defined(ARDUINO_SAMD_MKRWAN1310)
  #include "Arduino_LPWANConnectionHandler.h"`
 I see that in the other source files the header was included outside the #ifdef, so in the other cases a statement like the one I removed was required. So I think it is not required for the LoRaConnectionHandler but if you want to have consistency between the implementations I can change.